windows/src/global/delphi/general/Keyman.System.CanonicalLanguageCodeUtils.pas

```diff
@@ -47,22 +51,22 @@ class function TCanonicalLanguageCodeUtils.FindBestTag(const Tag: string; AddReg
       t.Tag := Result;
     end;
 
-    if not TLangTagsMap.LangTags.TryGetValue(t.Language, LangTag) then
+    if not TLangTagsMap.LangTags.TryGetValue(t.Tag, LangTag) then
```

Comment:
@ermshiperete changes to algorithm

Reply:
Nevermind, I think my build tool that creates the map is wrong. It doesn't process `tags`.